Project/Assets/ML-Agents/Examples/Basic/Scripts/BasicController.cs

```diff
@@ -72,10 +77,10 @@ public void ApplyAction(float[] vectorAction)
     }
 
     public void ResetAgent()
-    {
-        m_Position = 10;
-        smallGoal.transform.position = new Vector3(k_SmallGoalPosition - 10f, 0f, 0f);
-        largeGoal.transform.position = new Vector3(k_LargeGoalPosition - 10f, 0f, 0f);
+    {   
+        SceneManager.LoadScene(SceneManager.GetActiveScene().name);
```

Comment:
Can you add a note that the `SceneManager.LoadScene` call is just for testing, and in general isn't something that users need (or want) to do when resetting a scene?

Reply:
I added a note.